docs/csv-export.md

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_position: 9
+---
+
+# CSV Export
+
+OpenCost provides the ability to export data in CSV format to a local file, Azure Blob Storage, AWS S3, or Google Cloud Storage. This feature allows you to archive and analyze your data outside of OpenCost.
```

Comment:
Nit: is it worth clarifying this is "cost allocation" data?

Reply:
> I see this just [merged to the deploy branch](https://github.com/opencost/opencost/commit/8b2589b7b2e0d6c57fd8b67768f57f2dc5c8c076), so it should be in 1.103. Let's hold off on merging until we release 1.103

I think we're planning on a special kubecon release, so that would likely be the right time to merge this.